src/__tests__/interactions-helper.js

```diff
@@ -84,6 +84,18 @@ export const typeText = async (element, text, waitForDebounceMs = 250) => {
   return result;
 };
 
+export const pressNavigationKey = async (command = NAVIGATIONS_COMMANDS.TAB, waitForDebounceMs = 250) => {
```

Comment:
personally I think that `waitForDebounceMs` should be 0 by default. I think most cases where we navigate we don't have a debounce. Usually when we navigate it's not like we trigger an input, which might require a debounce... but I might be wrong, you're more familiar than me with the components :)